### ISSUE TITLE
asciidoc-preview install fail

### ISSUE THREAD

**Issue by Geekmano** [asciidoctor/atom-asciidoc-preview#110]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.19
**System**: Unknown Windows Version
**Thrown From**: [asciidoc-preview](https://github.com/asciidoctor/atom-asciidoc-preview) package, v0.6.0

### Stack Trace

Failed to load the asciidoc-preview package

```
At Cannot find module 'mustache'

Error: Cannot find module 'mustache'
  at Module._resolveFilename (module.js:334:15)
  at Function.Module._resolveFilename (D:\Atom\resources\app.asar\src\module-cache.js:383:52)
  at Function.Module._load (module.js:284:25)
  at Module.require (module.js:363:17)
  at require (module.js:382:17)
  at Object.<anonymous> (C:\Users\乐乐\.atom\packages\asciidoc-preview\lib\asciidoc-preview-view.coffee:6:12)
  at Object.<anonymous> (C:\Users\乐乐\.atom\packages\asciidoc-preview\lib\asciidoc-preview-view.coffee:1:1)
  at Module._compile (module.js:428:26)
  at Object.defineProperty.value [as .coffee] (D:\Atom\resources\app.asar\src\compile-cache.js:169:21)
  at Module.load (module.js:353:32)
  at Function.Module._load (module.js:308:12)
  at Module.require (module.js:363:17)
  at require (module.js:382:17)
  at Object.<anonymous> (C:\Users\乐乐\.atom\packages\asciidoc-preview\lib\asciidoc-preview.coffee:3:23)
  at Object.<anonymous> (C:\Users\乐乐\.atom\packages\asciidoc-preview\lib\asciidoc-preview.coffee:1:1)
  at Module._compile (module.js:428:26)
  at Object.defineProperty.value [as .coffee] (D:\Atom\resources\app.asar\src\compile-cache.js:169:21)
  at Module.load (module.js:353:32)
  at Function.Module._load (module.js:308:12)
  at Module.require (module.js:363:17)
  at require (module.js:382:17)
  at Package.module.exports.Package.requireMainModule (D:\Atom\resources\app.asar\src\package.js:761:34)
  at D:\Atom\resources\app.asar\src\package.js:192:28
  at Package.module.exports.Package.measure (D:\Atom\resources\app.asar\src\package.js:169:15)
  at Package.module.exports.Package.load (D:\Atom\resources\app.asar\src\package.js:183:12)
  at PackageManager.module.exports.PackageManager.loadPackage (D:\Atom\resources\app.asar\src\package-manager.js:405:14)
  at PackageManager.module.exports.PackageManager.loadPackages (D:\Atom\resources\app.asar\src\package-manager.js:369:14)
  at Atom.module.exports.Atom.startEditorWindow (D:\Atom\resources\app.asar\src\atom.js:667:21)
  at Object.<anonymous> (D:\Atom\resources\app.asar\src\window-bootstrap.js:12:8)
  at Object.<anonymous> (D:\Atom\resources\app.asar\src\window-bootstrap.js:23:4)
  at Module._compile (module.js:428:26)
  at Object.defineProperty.value [as .js] (D:\Atom\resources\app.asar\src\compile-cache.js:169:21)
  at Module.load (module.js:353:32)
  at Function.Module._load (module.js:308:12)
  at Module.require (module.js:363:17)
  at require (module.js:382:17)
  at setupWindow (file:///D:/Atom/resources/app.asar/static/index.js:82:25)
  at window.onload (file:///D:/Atom/resources/app.asar/static/index.js:35:9)

```

### Commands

```
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
asciidoc-preview, v0.6.0

# Dev
No dev packages
```

**Comment by ldez** [comment]
Can you retry with the 1.0.0 ?

**Comment by ldez** [comment]
Fixed in 1.0.0
